recipes/compute_library/all/conanfile.py

```diff
@@ -99,6 +101,10 @@ def validate(self):
         if self.settings.os == "Linux" and self.settings.compiler == "clang":
             # INFO: https://arm-software.github.io/ComputeLibrary/latest/how_to_build.xhtml#S1_2_linux
             raise ConanInvalidConfiguration(f"{self.ref} does not support Linux with clang. It is only supported on Linux with gcc.")
+        if "armv8" not in str(self.settings.arch) and self.options.enable_multi_isa:
+            raise ConanInvalidConfiguration(f"{self.ref} does not support multi_isa option for {self.settings.arch}. It is only supported on armv8.")
+        if self.settings.arch == "armv8" and self.build_settings.arch == "x86_64" and self.settings.os == "Macos":
+            raise ConanInvalidConfiguration(f"Mac Intel is not supported for armv8-a. Please, use Mac M1 as native build.")
```

Comment:
actually, cross compilation from macOS arm64 to x86_64 (as vice versa) works well. You need to use `-arch <arm64 | x86_64>` option.

See https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/thirdparty/ACLConfig.cmake#L285-L288 as example

Reply:
The problem is: I'm scons and it changes to 'armv8-a' that's not identified by apple clang and passing as -march.

Reply:
I also use scons, but it is just run from cmake.